File: app/src/main/java/com/shuoye/video/MainActivity.java
package com.shuoye.video;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Context;
import android.os.Bundle;

import com.shuoye.video.utils.Utils;

import static com.shuoye.video.utils.Utils.getContext;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Utils.init(this);
        setContentView(R.layout.activity_main);
    }

}

File: app/src/main/java/com/shuoye/video/ui/MainActivity.kt
package com.shuoye.video.ui

import android.annotation.SuppressLint
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.databinding.DataBindingUtil.setContentView
import com.shuoye.video.R
import com.shuoye.video.databinding.ActivityMainBinding
import com.shuoye.video.utils.StatusBarUtil
import dagger.hilt.android.AndroidEntryPoint

@AndroidEntryPoint
class MainActivity : AppCompatActivity() {
    @SuppressLint("ResourceAsColor")
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView<ActivityMainBinding>(this, R.layout.activity_main)
        // 设置状态栏颜色
        StatusBarUtil.setStatusBarMode(this, true, R.color.white)
    }
}